app/javascript/controllers/validations_controller.js

```diff
@@ -0,0 +1,66 @@
+import Rails from '@rails/ujs'
+import Turbolinks from 'turbolinks'
+import debounce from 'lodash.debounce'
+import ApplicationController from './application_controller'
+
+export default class extends ApplicationController {
+  static targets = ['firstName', 'lastName', 'email', 'submit']
+
+  connect () {
+    super.connect()
+    this.perform = debounce(this._perform, 250)
```

Comment:
Super useful, but does it belong in core? I don't think so, but I'm listening.

Reply:
We'd need to include something like `lodash.debounce` in core but then we could support both `stimuluate` and `debouncedStimuluate` out of the box. I'm not 100% certain I like the idea yet. Just thinking out loud.

Reply:
I do recommend that you switch over to importing `lodash-es` so that you can pull in exactly what you need without adding extra packages. This version designed for webpack tree shaking.